From 2a8dd80b5640d85771dac5b2b95a85321e5ecdf1 Mon Sep 17 00:00:00 2001
From: John Wass <jwass3@gmail.com>
Date: Mon, 14 Jan 2019 12:08:13 -0500
Subject: [PATCH] fix exporter

---
 research/object_detection/exporter.py | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/research/object_detection/exporter.py b/research/object_detection/exporter.py
index ef8fe19..95469e6 100644
--- a/research/object_detection/exporter.py
+++ b/research/object_detection/exporter.py
@@ -68,8 +68,7 @@ def freeze_graph_with_def_protos(
 
     if optimize_graph:
       logging.info('Graph Rewriter optimizations enabled')
-      rewrite_options = rewriter_config_pb2.RewriterConfig(
-          layout_optimizer=rewriter_config_pb2.RewriterConfig.ON)
+      rewrite_options = rewriter_config_pb2.RewriterConfig()
       rewrite_options.optimizers.append('pruning')
       rewrite_options.optimizers.append('constfold')
       rewrite_options.optimizers.append('layout')
-- 
2.7.4

